README.md

```diff
@@ -53,31 +57,39 @@ and replace `xxxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx` with the resource UUID you r
 Below are instructions for running a sample in each language included in this repository.
 
 #### How to run Node samples
+
 Using Node 16 or greater:
+
 1. `cd Node/`
-2. `npm install node-fetch` (need only run once)
+2. `npm install node-fetch`
```

Comment:
This is the only change I can identify that did not pertain to formatting -- is that accurate?

Reply:
This is correct.